app/build.gradle.kts

```diff
@@ -0,0 +1,70 @@
+plugins {
+    id("com.android.application")
+    id("org.jetbrains.kotlin.android")
+}
+
+android {
+    namespace = "com.example.android.picover"
+    compileSdk = 33
+
+    defaultConfig {
+        applicationId = "com.example.android.picover"
+        minSdk = 26
+        targetSdk = 33
+        versionCode = 1
+        versionName = "1.0"
+
+        testInstrumentationRunner = "androidx.test.runner.AndroidJUnitRunner"
+        vectorDrawables {
+            useSupportLibrary = true
```

Comment:
is it needed if we have `minSdk = 26`?

Reply:
I didn't consider it because the scope of these changes only migrates `Gradle` to `Gradle Kotlin`.

Reply:
I know :) but you touch the whole file, so you can do some cleaning at once ;)

Reply:
I will delete it :D

Reply:
DONE